src/permissionEntryRow.ui

```diff
@@ -26,7 +26,7 @@
                 <property name="visible">True</property>
                 <property name="can_focus">False</property>
                 <property name="halign">start</property>
-                <property name="label" translatable="yes">Pemission</property>
+                <property name="label" translatable="yes">Permission</property>
```

Comment:
Thanks for catching those. Note that these strings are not user-facing, so additionally "translateable" we must remove these "translateable" tags, so the string can be remove (automatically) from the .pot and .po files when I updated them.

Reply:
Done. :-)